### PR TITLE
Improve scripting and others

### PR DESCRIPTION
**Pull request by gpetit**
- Add a "Map composer" part in the page "Cartography"
- "Show attributes" page has been renamed to "Manage Data". Tables informations are now in a sub-part
- Some images in the folder "_images" have been reorganized into sub-folders to be more clear
- Add a page for "Plugins"
- Create "Scripting" pages for SQL / Groovy / R languages
- SQL page has been improved with use cases

### CODE REVIEW

File: docs/users/cartography.rst [hunk 1]
```diff
@@ -269,26 +269,36 @@ Using the toolbar, the user can:
...
-.. |ToolDown| image:: ../_images/thematic_tool_down.png
+.. |ToolDown| image:: ../_images/cartography/thematic_tool_down.png
               :alt: Down icon
               :width: 16 pt
 
 
+Compose & export maps
+-------------------------------------------
+
+With the `Map Composer`_ plugin, user can compose maps. This interface gives users the opportunity to add text, orientation, scale, \... and to export the result in .pdf or image files. A detailled documentation is provided `HERE`_.
```
Comment (ebocher) [inline docs/users/cartography.rst]: to create map layout

Comment (ebocher) [reply docs/users/cartography.rst]: A map layout is a collection of map elements organized on a page and designed for map printing. With The Map Composer you can add map elements such  text, orientation, scale. Once you have completed a map layout, you can print it or export it  of output formats as   .pdf or image file

File: docs/users/manage_data.rst [hunk 2]
```diff
@@ -1,13 +1,21 @@
 .. Author : Gwendall PETIT (Lab-STICC - CNRS UMR 6285 / DECIDE Team)
 
+
+.. _manage_data:
+
+Manage data
+=====================
+
```
Comment (ebocher) [inline docs/users/manage_data.rst:8]: As other GIS plateform, OrbisGIS offers features to manipulate data.

File: docs/users/plugins.rst [hunk 3]
```diff
@@ -0,0 +1,153 @@
+.. Author : Gwendall PETIT (Lab-STICC - CNRS UMR 6285 / DECIDE Team)
+
+.. _plugins:
+
+Plugins 
+=================
+
+OrbisGIS is able to extent its functionnalities through the use of plugins.
```
Comment (ebocher) [inline docs/users/plugins.rst]: OrbisGIS could be extended with new features thanks to its plugin manager.

File: docs/users/plugins.rst [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+              :alt: Uninstall the plugin
+	      :width: 16 pt
+
+
+
+
+Official plugins
+------------------------------------
+
+In addition of already provided *(by default)* plugins, user can optionnaly add some official OrbisGIS plugins, using the `Plugins manager`_.
```
Comment (ebocher) [inline docs/users/plugins.rst]: Put this sentence on top of the page and just explain official plugins repository.

File: docs/users/plugins.rst [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+.. _ChartView: https://github.com/orbisgis/orbisgis-official-plugins/tree/master/chartView
+.. _Groovy Console: https://github.com/orbisgis/orbisgis-official-plugins/tree/master/groovyConsole
+.. _Mailto: https://github.com/orbisgis/orbisgis-official-plugins/tree/master/mailto
+.. _Map Composer: https://github.com/orbisgis/orbisgis-official-plugins/tree/master/mapComposer
+.. _JVM memory: https://github.com/orbisgis/orbisgis-official-plugins/tree/master/memory
+.. _R Console: https://github.com/orbisgis/orbisgis-official-plugins/tree/master/r-console
+
+
+
```
Comment (ebocher) [inline docs/users/plugins.rst]: Remove empty lines

File: docs/users/cartography.rst (hunk 1 above)
Comment (gpetit) [reply docs/users/cartography.rst]: Ok

File: docs/users/manage_data.rst (hunk 2 above)
Comment (gpetit) [reply docs/users/manage_data.rst:8]: Ok

File: docs/users/plugins.rst (hunk 3 above)
Comment (gpetit) [reply docs/users/plugins.rst]: ok

File: docs/users/plugins.rst (hunk 4 above)
Comment (gpetit) [reply docs/users/plugins.rst]: ok

File: docs/users/plugins.rst (hunk 5 above)
Comment (gpetit) [reply docs/users/plugins.rst]: ok

**Pull request merged**